src/pages/layer-2.js

```diff
@@ -0,0 +1,614 @@
+      {
+        content: "Wait, what's Ethereum?",
+        pathId: "/",
+        isSecondary: "isSecondary",
+      },
+    ],
+  }
+
+  const layer2Cards = [
+    {
+      image: privacyGlyph,
+      title: "Inclusivity",
+      preview: "Lower fees and better user experience",
+      body: [
+        "By combining multiple transactions into a single transaction on layer 1, transaction fees are massively reduced, making Ethereum more accessible for all.",
+      ],
+      alt: "test",
+    },
+    {
+      image: privacyGlyph,
+      title: "Expand Use Cases",
+      preview: "Increase the scope of projects",
+      body: [
+        "With higher transactions per second, cheaper fees, and new technology, the scope of projects is able to expand into new use cases we have yet to see come online. Opportunities for gaming, exchanges, and applications yet to be created will be able to come online with better user experience.",
+      ],
+      alt: "test",
+    },
+    {
+      image: privacyGlyph,
+      title: "Security",
+      preview: "Layer 2 is cashing a cheque, layer 1 is going to court",
+      body: [
+        "We need need scaling solutions that inherit Ethereums security. Layer 2 networks settle their transactions on the Ethereum mainnet, allowing users who use them to benefit from the security of the Ethereum network.",
+        "Layer 2's take advantage of the consensus mechanism of its parent Layer 1, in this case Ethereum, instead of providing their own consensus mechanism.",
+      ],
+      alt: "test",
+    },
+  ]
+
+  const layer2DataCombined = [...layer2Data.optimistic, ...layer2Data.zk]
+
+  return (
+    <GappedPage>
+      <PageMetadata
+        title={"Layer 2"}
+        description={"Introduction page to layer 2"}
+      />
+
+      <HeroContainer>
+        <Hero content={heroContent} isReverse />
+      </HeroContainer>
+
+      <Content>
+        <h2>What is Layer 1?</h2>
+        <p>
+          Layer 1 blockchains, like Ethereum and Bitcoin, are the underlying
+          foundation that layer 2 projects build on top of. Some examples of
+          layer 2 projects being build on top of these networks are zk-rollups
+          and optimistic-rollups on Ethereum, and the Lighting Network on top of
+          Bitcoin.
+        </p>
+        <p>
+          Ethereum as the layer 1 includes the blockchain itself and the history
+          of transaction data, a network of miners (soon to be validators after{" "}
+          <Link to="/upgrades/merge/">The Merge</Link>) proposing blocks, a
+          network of node operators to secure and validate the network, and the
+          consensus mechanism for the network. Ethereum also function as a data
+          availability layer for Layer 2's, and if there are any disputes on
+          previous transactions data is provided from Ethereum for these
+          disputes.
+        </p>
+        <p>
+          While Ethereum has been a breakthrough technology in smart contract
```

Comment:
> While Ethereum has been a breakthrough technology in smart contract blockchains, it faces the same trilemma between scaling, decentralization, and security as any other blockchain.

Again might be too much jargon right off the bat here. Should we expect people to know the scalability trilemma? I think we could put this more simply that demand for Ethereum is higher than current capacity. It needs to scale while maintaining security & decentralization, 2 core tenants of what makes a blockchain valuable.

Reply:
Overall I think this paragraph can be tightened up a bit.

> While there are roadmap plans to improve the scalability of Ethereum, there is still a need for further scaling to round out the trilemma for Ethereum.

We could maybe drop this entire sentence, we've pretty much already covered these points.

> As time has progressed, Ethereum as a layer 1 is a settlement layer for layer 2s, and not the layer for end users.

Important point but not sure this would make sense for someone just learning about this. It's not yet clear that L2s "use" L1 as a settlement layer by submitting bundled transactions to the L1. Perhaps we save this until later on when it would be clearer.